package.json

```diff
@@ -3,7 +3,7 @@
   "author": "NuoDB, Inc.",
   "description": "The official NuoDB driver for Node.js. Provides a high-level SQL API on top of the NuoDB Node.js Addon.",
   "license": "Apache-2.0",
-  "version": "3.2.3",
```

Comment:
I think my last commit increased the patch number by one, but I think I am ok with your change moving the minor number, just the diffs will be slightly different.